openedx/features/course_experience/course_tools.py

```diff
@@ -3,12 +3,22 @@
 """
 
 
+from enum import Enum
+
 from openedx.core.lib.plugins import PluginManager
 
 # Stevedore extension point namespace
 COURSE_TOOLS_NAMESPACE = 'openedx.course_tool'
 
 
+class HttpMethod(Enum):
```

Comment:
I woud expect this is already in a package we rely on... But doesn't seem to be in `requests` anyway.

Reply:
Yeah, I couldn't find it so I did this so it is at least somewhere and we can potentially move it around if other apps want it.